modules/frontend/tenant.go

```diff
@@ -1,173 +1,19 @@
 package frontend
 
 import (
-	"context"
 	"errors"
 	"io"
 	"net/http"
-	"strconv"
 	"strings"
-	"sync"
 
 	"github.com/go-kit/log"
 	"github.com/go-kit/log/level"
 	"github.com/grafana/dskit/tenant"
-	"github.com/grafana/dskit/user"
-	"github.com/grafana/tempo/modules/frontend/combiner"
 	"github.com/grafana/tempo/modules/frontend/pipeline"
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promauto"
-	"github.com/uber-go/atomic"
-)
-
-const (
-	statusCodeLabel = "status_code"
-	tenantLabel     = "tenant"
 )
 
 var ErrMultiTenantUnsupported = errors.New("multi-tenant query unsupported")
 
-var (
-	tenantSuccessTotal = promauto.NewCounterVec(
```

Comment:
looks like we removing these metrics, can we mention it as a breaking change. this might break dashboards and alerts for end users.

Reply:
yup, i am going to create a big changelog PR when i'm done and will include this detail